### DISCUSSION TITLE
Override node-engine

### DISCUSSION THREAD

**Discussion by jedjebari** [paketo-buildpacks/feedback#299] [Node.js Team]
Hello
At this moment the latets nodejs buildpack v2.2.0 uses node-engine v3.2.1.
Is there a way for customers to override this to use a higher version of node-engine ?
Thanks
Jerome

**Comment by sophiewigmore** [comment]
Hey there - I just pinged the Node.js maintainers to see if they can work on a Node.js buildpack release to get this updated. In the meantime, yes, you can overwrite the buildpack version yourself. Here's an example:

let's say you're building like this now, and for example it pulls in Node Engine, Yarn, and Yarn Install:
`pack build <app> -b paketo-buildpacks/nodejs`
You can specify versions to use like this:
`pack build <app> -b paketo-buildpacks/node-engine@<VERSION> -b paketo-buildpacks/nodejs`
